src/v2/Apps/Home/Components/HomeArtworkModuleRail.tsx

```diff
@@ -24,8 +24,8 @@ const HOME_ARTWORK_MODULES = [
   "FOLLOWED_ARTIST",
   "FOLLOWED_ARTISTS",
   "FOLLOWED_GALLERIES",
-  "GENERIC_GENES",
-  "FOLLOWED_GENES",
+  "GENERIC_GENES", // GENERIC_GENES
+  "FOLLOWED_GENES", // FOLLOWED_GENES
```

Comment:
What's with the comments?

Reply:
looks like holdovers from before I updated the types in MP; will fix